src/map/itemdb.c

```diff
@@ -2079,7 +2079,7 @@ int itemdb_readdb_libconfig_sub(struct config_setting_t *it, int n, const char *
 	if (itemdb->lookup_const(it, "ViewSprite", &i32) && i32 >= 0)
 		id.view_sprite = i32;
 
-	if (itemdb->lookup_const(it, "View", &i32) && i32 >= 0) { // TODO: Remove (Deprecated - 2016-09-04 [Haru])
+	if (itemdb->lookup_const(it, "View", &i32) && i32 >= 0) { // TODO: Remove (Deprecated - 2018-09-04 [Haru])
```

Comment:
I believe this is not an intended changes.
try not to search and replace using the sole keyword `2016`.

Reply:
wait pls

Reply:
Fixed : 5369c79